deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py

```diff
@@ -295,3 +295,11 @@ def check_refresh_feed_log(provider_name):
     check_vuln_detector_event(timeout=vd.VULN_DETECTOR_GLOBAL_TIMEOUT,
                               callback=f"Refresh of '{provider_name}' database finished",
                               error_message=f"Could not find {provider_name} feed refresh log")
+
+
+def check_configuration_error():
+    """Check the configuration error event in ossec.log"""
+    check_vuln_detector_event(timeout=vd.DOWNLOAD_TIMEOUT,
+                              callback=r".* \(\d+\): Configuration error at.*",
+                              error_message=f"Could not find the event 'Configuration error at 'etc/ossec.conf' "
+                                             'in ossec.log', prefix='.*wazuh-modulesd.*')
```

Comment:
```suggestion
                              error_message=f"Could not find the event 'Configuration error at 'etc/ossec.conf' "
                                            'in ossec.log', prefix='.*wazuh-modulesd.*')
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2547/commits/02e46332a8d4b6f7aacf789e3157df9457812718